api/models/graphite_gen.go

```diff
@@ -372,6 +372,356 @@ func (z *GraphiteTagDelSeriesResp) Msgsize() (s int) {
 	return
 }
 
+// DecodeMsg implements msgp.Decodable
```

Comment:
why are we generating all the msgp code, tests and benchmarks? we only use json, so let's add an ignore rule for this type.